package.json

```diff
@@ -2062,11 +2062,12 @@
         "lint": "eslint -c .eslintrc.js --ext .ts .",
         "lintfix": "eslint -c .eslintrc.js --fix --ext .ts .",
         "package": "ts-node ./build-scripts/package.ts",
-        "install-plugin": "vsce package -o aws-toolkit-vscode-test.vsix && code --install-extension aws-toolkit-vscode-test.vsix",
```

Comment:
`install-plugin` didn't do compile before, are you sure we should add any compile to it?

Reply:
Yeah I'll fix the webpacking so that it's part of `compile`